[rospy.client][INFO] 2019-06-25 13:22:35,942: init_node, name[/rostopic_71094_1561468955822], pid[71094]
[xmlrpc][INFO] 2019-06-25 13:22:35,943: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-25 13:22:35,943: Started XML-RPC server [http://dc5b429f784e:44995/]
[rospy.init][INFO] 2019-06-25 13:22:35,944: ROS Slave URI: [http://dc5b429f784e:44995/]
[rospy.impl.masterslave][INFO] 2019-06-25 13:22:35,944: _ready: http://dc5b429f784e:44995/
[rospy.registration][INFO] 2019-06-25 13:22:35,945: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2019-06-25 13:22:35,946: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-06-25 13:22:36,044: registered with master
[rospy.rosout][INFO] 2019-06-25 13:22:36,045: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 13:22:36,062: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 13:22:36,064: initializing /clock core topic
[rospy.simtime][INFO] 2019-06-25 13:22:36,081: connected to core topic /clock
[rospy.internal][INFO] 2019-06-25 13:22:36,122: topic[/scan] adding connection to [http://dc5b429f784e:35387/], count 0
[rospy.internal][INFO] 2019-06-25 13:22:36,124: topic[/clock] adding connection to [http://dc5b429f784e:35387/], count 0
[rospy.internal][INFO] 2019-06-25 13:22:36,294: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2019-06-25 13:22:36,985: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 13:22:36,995: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-06-25 13:22:36,995: topic[/clock] removing connection to http://dc5b429f784e:35387/
[rospy.internal][ERROR] 2019-06-25 13:22:36,996: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 729, in receive_once
    self.stat_bytes += recv_buff(sock, b, p.buff_size)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 103, in recv_buff
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2019-06-25 13:22:36,997: topic[/scan] removing connection to http://dc5b429f784e:35387/
[rospy.impl.masterslave][INFO] 2019-06-25 13:22:36,997: signal-2
[rospy.internal][WARNING] 2019-06-25 13:22:36,998: Unknown error initiating TCP/IP socket to dc5b429f784e:52421 (http://dc5b429f784e:35387/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2019-06-25 13:22:37,508: signal_shutdown [signal-2]
[rospy.core][INFO] 2019-06-25 13:22:37,508: signal_shutdown [atexit]
